code/__defines/experience.dm

```diff
@@ -48,4 +48,6 @@
 #define SKILL_MAGIC_OFFENSIVE "offensive" //Completed
 #define SKILL_MAGIC_SUMMONING "summoning" //Completed
 #define SKILL_MAGIC_ENCHANTING "enchanting" //Completed.
-#define SKILL_MEDICINE "medicine" //Completed
\ No newline at end of file
+#define SKILL_MEDICINE "medicine" //Completed
+
```

Comment:
Rather certain that this line skip is unnecesary.

Reply:
It is helpful, since Botany isn't attached to a stat the way the existing 15 skills are organized. I would keep Botany in its own no skill attached category until there's 2 more skills, so the STR/DEX/INT skill symmetry can be retained. Comments will be added to clarify this.